### DISCUSSION TITLE
How to add custom build folder?

### DISCUSSION THREAD

**Discussion by RainyPixel** [parcel-bundler/parcel#5059] [General]
I use only package.json to "configure" parcel. Version - nightly. Then i use command: `parcel build ./src/index.html --dist-dir dist` they build all files to my root folder :D. Thanks for response!

**Comment by cupcakearmy** [comment]
You mean this maybe?

```
-d, --out-dir <path>           set the output directory. defaults to "dist"
```

**Comment by mischnic** [comment]
I think this has been resolved here: https://github.com/parcel-bundler/parcel/issues/5063#issuecomment-679304057
